src/steps/mdx-to-gridtables.ts

```diff
@@ -106,23 +106,26 @@ export default function mdxToBlocks(ctx: Helix.UniversalContext) {
                 ],
               },
             ],
-          },{
-            type: "paragraph",
-            children: [
-              {
-                type: "strong",
-                children: [
-                  {
-                    type: "text",
-                    value: (node.attributes || [])
-                    .filter(isMdxJsxAttribute)
-                    .filter((attribute) => attribute.name !== "slots")
-                    .map((attribute) => `${ATTRIBUTE_PREFIX}${attribute.name}=${attribute.value}`)
-                  },
-                ],
-              },
-            ],
           },
+          ...(node.attributes && node.attributes.some(attr => attr.name !== "slots")
```

Comment:
just curious, why are we destructuring here (...) when the expression already evaluates to a boolean?